### PR TITLE
fix(quickPreview): fixes Assignee/label dropdown issue

### PR DESCRIPTION
**Pull request by sahil143**
### [ ] What does this PR do? _[Mandatory]_
fixes 7,8 
> Add label, assignees can open simultaneously, there should be only one area of focus. 
> Add label/Add assignees > no action but yet we get a notification

in https://docs.google.com/document/d/1SjzX_j6ZaEApGoY0gKgnvU_FqpNT6M3uu6O7JeUTfrc/edit

### [ ] What issue/task does this PR references? _[Mandatory]_

OSIO issues
- https://github.com/openshiftio/openshift.io/issues/2736
- https://github.com/openshiftio/openshift.io/issues/2642

### [ ] Are the tests Included? _[Mandatory]_

Yes

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
[test]

**Comment by pranavgore09** [comment]
[test]

**Comment by rgarg1** [comment]
New changes observed:
- [ ] Assignees and Labels drop-downs now close on clicking outside the modal
- [ ] Closing the modal by clicking outside preserves the changes made to the drop-down selection
- [ ] Only one drop-down can now remain active at any time
- [ ] Unintentional edits on opening and closing drop-downs are no longer happening

**Comment by rgarg1** [comment]
@sahil143 @nimishamukherjee @michaelkleinhenz The current behavior looks good to QE. However, if UX wants to see this change further, we shall track that change separately. The current UX matches the experience in GH too.

**Comment by rgarg1** [comment]
@Raunak1203 Please evaluate an impact of these changes on existing tests.
@sahil143 Can we please add a few tests too.

File: src/tests/specs/quickPreview.spec.ts [hunk 1]
```diff
@@ -58,4 +58,12 @@ describe('Quick preview tests: ', () => {
     // The description box should not be in edit mode
     expect(await planner.quickPreview.isSaveButtonDisplayed()).toBeFalsy();
   })
+
+  it('should close assignee dropdown when clicked outside',async () => {
+    await planner.workItemList.clickWorkItem(c.workItemTitle1);
+    await planner.quickPreview.assigneeDropdown.clickWhenReady();
+    expect(await planner.quickPreview.assigneeDropdownMenu).toContain('show');
```
Comment (Raunak1203) [inline src/tests/specs/quickPreview.spec.ts]: please change expect(await planner.quickPreview.assigneeDropdownMenu).toContain('show');
to expect(await planner.quickPreview.assigneeDropdownMenu.getAttribute('attrName).toContain('show'); test is failing

File: src/tests/specs/quickPreview.spec.ts [hunk 2]
```diff
@@ -58,4 +58,12 @@ describe('Quick preview tests: ', () => {
...
+    await planner.quickPreview.titleInput.clickWhenReady();
+    expect(await planner.quickPreview.assigneeDropdownMenu).not.toContain('show');
```
Comment (Raunak1203) [inline src/tests/specs/quickPreview.spec.ts]: same as above

File: src/tests/specs/quickPreview.spec.ts [hunk 3]
```diff
@@ -58,4 +58,12 @@ describe('Quick preview tests: ', () => {
     // The description box should not be in edit mode
     expect(await planner.quickPreview.isSaveButtonDisplayed()).toBeFalsy();
   })
+
+  fit('should close assignee dropdown when clicked outside',async () => {
```
Comment (Raunak1203) [inline src/tests/specs/quickPreview.spec.ts]: @sahil143 please remove fit

**Comment by sahil143** [comment]
#2596 
Raised another Pr only for "close dropdown when clicked outside "
> Add label/Add assignees > no action but yet we get a notification
https://github.com/openshiftio/openshift.io/issues/2642

This will be fixed in another PR

**Pull request closed without merging**